v2/devdoc/log_context_property_type_struct_requirements.md

```diff
@@ -0,0 +1,85 @@
+# `log_context_property_type_struct` requirements
+
+`log_context_property_type_struct` implements the code needed for handling a structure that contains several fields.
```

Comment:
>handling a structure [](http://example.com/codeflow?start=66&length=20)

For the life of me if I can see any struct.

What I see is "number of fields" that gets initialized/copy'd etc.

Reply:
Also there's no reference to "struct" and what it does/doesn't in logging.md. :(((

Reply:
This is meant to be used internally to save the name of the context and the field count for a context